controllers/profiles/getOtherFeedsInProfile.js

```diff
@@ -77,6 +59,8 @@ module.exports = async (req, res) => {
         if (now < dateExpired || item.duration_feed == 'never') {
           let newItem = {...item};
           if (item.anonimity) continue;
+
+          newItem.is_following_target = userFollow ? true : false;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for setting `newItem.is_following_target` seems to be correct, but it's not clear where the `userFollow` variable is defined. If it's not defined in the code above this hunk, there might be a reference error. Please ensure that `userFollow` is properly defined and its value is correctly set before this line.
```diff
- newItem.is_following_target = userFollow ? true : false;
+ // Assuming userFollow is defined somewhere above in the code
+ newItem.is_following_target = userFollow ? true : false;
```
If `userFollow` is a boolean, you don't need the ternary operator; you can just assign its value directly to `newItem.is_following_target`.
```diff
- newItem.is_following_target = userFollow ? true : false;
+ newItem.is_following_target = userFollow;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->